app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -128,7 +130,7 @@ ManageIQ.angularApplication.controller('emsCommonFormController', ['$http', '$sc
        $scope.emsCommonModel.default_password != '' && $scope.angularForm.default_password.$valid &&
        $scope.emsCommonModel.default_verify != '' && $scope.angularForm.default_verify.$valid)) {
       return true;
-    } else if($scope.emsCommonModel.emstype == "gce") {
+    } else if($scope.emsCommonModel.emstype == "gce" && $scope.emsCommonModel.project != '') {
       return true;
```

Comment:
This conditional decides if ```Validate``` button will be active or not. Make sure you include *all* required conditions.

```javascript
$scope.currentTab == "service_account" 
$scope.emsCommonModel.emstype == "gce"
$scope.emsCommonModel.project != '' && $scope.angularForm.project.$valid
$scope.emsCommonModel.service_account_json != ''
```